filter.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <title>DesignWays Hub - edukacyjny hub dla designerów</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js"></script>
```

Comment:
nie w headzie - na bottom nad Twój plik jedynie :)

Reply:
Błagam tylko nie moment.js to najwieksza [kobyła](https://bundlephobia.com/result?p=moment@2.29.1) 😜 [dayjs](https://day.js.org/) jest chyba najmniejszy